app/helpers/form_helper.rb

```diff
@@ -59,6 +59,8 @@ def scoped_parent_activities(activity:, user:)
       FindProjectActivities.new(organisation: activity.organisation, user: user)
         .call(eager_load_parent: false)
```

Comment:
Out of curiosity... Is this `eager_load_parent: false` to avoid errors from the Bullet gem or is there any other reason?